app/gui/src/project-view/components/FunctionSignatureEditor.vue

```diff
@@ -77,13 +77,17 @@ const rootStyle = computed(() => {
       groupBasedColor.value ?? returnTypeBasedColor.value ?? 'var(--group-color-fallback)',
   }
 })
+
+// We surely don’t have primary application for the function definition.
+const primaryApplication = { function: null, accessChain: null, potentialSelfArgument: null }
```

Comment:
anther usage of `emptyPrimaryApplication` function.

Or we should consider just making this field nullable? It shouldn't be bad, we could just add `?` to most usages.

Reply:
Done. It can’t be nullable for reactivity reasons, as pointed out by Kaz in this thread: https://github.com/enso-org/enso/pull/12751#discussion_r2050377740